app/scripts/modules/tencentcloud/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@spinnaker/tencentcloud",
```

Comment:
Don't forget to include the version in this file.

Reply:
> Don't forget to include the package version in this file.

Thanks for your guide. 
But if I add version field in the package.json, I will not pass the actions, cuz the previous version is empty, it's not equal to 0.0.1, how can I fix it ? Do I need to rewrite assert_package_bumps_standalone.sh or ... ?
![image](https://user-images.githubusercontent.com/15920861/81134249-beefd600-8f86-11ea-857b-f2559fc89dc8.png)
https://github.com/spinnaker/deck/runs/614469005?check_suite_focus=true

Reply:
> Don't forget to include the package version in this file.

Thanks for your guide. 
But if I add version field in the package.json, I will not pass the actions, cuz the changes in package.json is not only version field but also includes the other fields, in addition, the previous version is empty. It cause that the version change is not only change in package.json and fail the action's pipeline. How could I fix it ? Do I need to rewrite assert_package_bumps_standalone.sh or ... ?
![image](https://user-images.githubusercontent.com/15920861/81134249-beefd600-8f86-11ea-857b-f2559fc89dc8.png)
https://github.com/spinnaker/deck/runs/614469005?check_suite_focus=true

Reply:
You can always merge this PR with version `0.0.0` . Then if the action fails, you can follow up with a simple PR that bumps the version to `0.0.1` so the action will be triggered.

Reply:
> You can always merge this PR with version `0.0.0` . Then if the action fails, you can follow up with a simple PR that bumps the version to `0.0.1` so the action will be triggered.

Thanks for your suggestion, I added version `0.0.0` to package.json

Reply:
If you keep running into build failures with this, you can also divide this into 2 PRs. The first would include all the module set up and the second would add the config for the workflow.

Reply:
> If you keep running into build failures with this, you can also divide this into 2 PRs. The first would include all the module set up and the second would add the config for the workflow.

As the first PR,  this time I remove the tencentcloud config for the workflow and version field in package.json. I will add the version field and config for the workflow in the next PR. Is it all right?  If not could you give me some suggestions with more detail?